server/bin/pbench-server-put-shim-001.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+# This script is used to prepare the tarballs that a version 002 client
+# submits for further processing. It copies the tarballs and their MD5
+# sums to the archive (after checking) and sets the state links, so
+# that the dispatch script will pick them up and get the ball
+# rolling.
+
+import os
+import sys
+import glob
+import tempfile
+
+from logging import Logger
+from pathlib import Path
+from collections import namedtuple
+
+from pbench.agent.results import CopyResultTb
+from pbench.common.exceptions import BadConfig
+from pbench.common.logger import get_pbench_logger
+from pbench.common.utils import md5sum
+from pbench.server import PbenchServerConfig
+from pbench.server.report import Report
+
+
+_NAME_ = "pbench-server-prep-shim-002"
+
+Results = namedtuple("Results", ["nstatus", "ntotal", "ntbs"])
+
+
+def get_receive_dir(config: PbenchServerConfig, logger: Logger) -> Path:
+
+    # we are explicitly handling version-002 data in this shim
+    receive_dir_prefix = config.get("pbench-server", "pbench-receive-dir-prefix")
```

Comment:
What is the significance of this code comment?  Does it indicate that we don't actually need to be fetching the prefix from the configuration file?  Or, does it mean that we should be applying a default if the option is missing from the file?  (Or, is the comment stale and should be removed?)

Reply:
seems like stale